### DISCUSSION TITLE
updateable?

### DISCUSSION THREAD

**Discussion by ChrisVonFrosta** [Jason2866/Portable_VSC_PlatformIO#6] [Q&A]
Hi, looks like i am looking for. But is that VSC Version simple updatable?
Is it possible to install more extension?

**Comment by Jason2866** [comment]
The project you want to compile can be updated any time. VSC can be updated if you copy the unpacked contents from the new downloaded portable VSC version to its correct places.  VSC extensions can be installed and used as in every other VSC installation.
If updating Platformio it will loose its portability. Does mean it works where it is currently installed (Drive and path). But not on a different drive nor path.

**Comment by Jason2866** [comment]
Updated VSC and Platformio to currently latest versions. See release section
